cpp/dolfinx/fem/assemble_matrix_impl.cpp

```diff
@@ -328,11 +360,17 @@ void fem::impl::assemble_interior_facets(
 
     // Get local index of facet with respect to the cell
     const int local_facet[2] = {cell0.index(facet), cell1.index(facet)};
-    const int orient[2] = {0, 0};
 
     // Get cell vertex coordinates
     const int cell_index0 = cell0.index();
     const int cell_index1 = cell1.index();
+
+    const std::uint8_t perm[2]
```

Comment:
Do we use pure C arrays somewhere else? This could be `const std::array<std::uint8_t, 2>` instead

Reply:
perm is passed into ffc. Changing this to a `std::array` is causing errors (unless I'm doing something stupid?)

Reply:
Sure, but you would need to pass `perm.data()` to get the underlying pointer. It would be more c++-ish. But I see now raw arrays at more places, I would prefer to see more std::arrays.

Reply:
I've replaced perm[2] and local_facet[2] with std:arrays